clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -330,6 +331,12 @@ synchronized public DescribeTopicsResult describeTopics(Collection<String> topic
                 future.completeExceptionally(new UnknownTopicOrPartitionException("Topic " + requestedTopic + " not found."));
                 topicDescriptions.put(requestedTopic, future);
             }
+            // try to simulate the leader not available situation when topic name is "LeaderNotAvailableTopic"
```

Comment:
This workaround is very hard to be found by other developers, as a minimum we should define a constant and make it part of `MockAdminClient` class

Reply:
Good suggestion. I put it as a constant. Thanks.